pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -0,0 +1,148 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2021 Datadog, Inc.
+
+//go:build kubeapiserver
+// +build kubeapiserver
+
+package helm
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	checkName               = "helm"
+	maximumWaitForAPIServer = 10 * time.Second
+	informerSyncTimeout     = 60 * time.Second
+)
+
+func init() {
+	core.RegisterCheck(checkName, factory)
+}
+
+// HelmCheck collects information about the Helm releases deployed in the
+// cluster. The check only works for Helm installations configured to use
+// Kubernetes secrets as the storage. K8s secrets are the default in Helm v3.
+// Helm v2 used config maps by default. Ref:
+// https://helm.sh/docs/faq/changes_since_helm2/#secrets-as-the-default-storage-driver
+type HelmCheck struct {
+	core.CheckBase
+	secretLister      v1.SecretLister
+	runLeaderElection bool
+}
+
+var helmSecretsSelector = labels.Set{"owner": "helm"}.AsSelector()
+
+func factory() check.Check {
+	return &HelmCheck{
+		CheckBase:         core.NewCheckBase(checkName),
+		runLeaderElection: !config.IsCLCRunner(),
```

Comment:
When I run this check on the DCA, it adds the `kubernetes_cluster_name` and `cluster_name` tags. However, I don't see those tags when the check runs in the check runners. Is this expected or am I missing something?

Reply:
It was because I was missing some configuration calls in the `Configure()` function.
I added a new commit that fixes this.